src/utils/transaction.js

```diff
@@ -172,10 +172,10 @@ const createTransactionObject = (tx, moduleAssetId) => {
 
     case unlockToken: {
       transaction.asset = {
-        unlockObjects: tx.unlockingObjects.map(unlockingObject => ({
-          amount: BigInt(unlockingObject.amount),
-          delegateAddress: getAddressFromBase32Address(unlockingObject.delegateAddress),
-          unvoteHeight: unlockingObject.unvoteHeight,
+        unlockObjects: tx.unlockObjects.map(unlockObject => ({
+          amount: BigInt(unlockObject.amount),
+          delegateAddress: getAddressFromBase32Address(unlockObject.delegateAddress),
+          unvoteHeight: unlockObject.unvoteHeight,
```

Comment:
Isn't this supposed to be unlockobject.height.start? This is what I remember but may be its changed now. Can you please confirm again?